snapcraft/cli/store.py

```diff
@@ -140,7 +141,7 @@ def push(snap_file, release):
         channel_list = release.split(',')
         click.echo(
             'After pushing, an attempt to release to {} '
-            'will be made'.format(channel_list))
+            'will be made'.format(humanize_list(channel_list, 'and')))
```

Comment:
While you're at it, mind rewording this slightly? `'After pushing, an attempt will be made to release to {}'` would be better, I think.